### PR TITLE
add missing property

### LINKED ISSUE #2114: Check new added/renamed properties consistent with doc
I am working on enabling auto-generation for tables in doc. I will migrate all properties from old doc to new doc and will try to find the renamed properties for each interface, and replace them in the doc. But I need help from people who added/renamed properties in those interfaces to put these new properties into the right table after I finish enable all the table. I have already check every properties for `Axis` and `Legend` and may need to check the following doc as well. And this is a list for all docs need to be checked.

- [x] bin.md
- [x] config.md
- [x] data.md
- [x] encoding.md
- [x] facet.md
- [x] label.md
- [x] mark.md
- [x] overview.md
- [x] scale.md
- [x] size.md
- [x] sort.md
- [x] spec.md
- [x] timeunit.md
- [x] transform.md

### PR DESCRIPTION
**Pull request by YuhanLu**
fix #2114 and part of #2362

### CODE REVIEW

File: site/docs/axis.md [hunk 1]
```diff
@@ -80,12 +80,12 @@ To provide themes for all axes, the axis config `config: {axis: {...}}` can cont
...
 
 ### Ticks
 
-{% include table.html props="tickColor,tickRound,labelColor,labelFont,labelFontSize,labelLimit,tickWidth" source="AxisConfig" %}
+{% include table.html props="ticks,tickColor,tickExtra,tickRound,tickSize,labelColor,labelFont,labelFontSize,labelLimit,tickWidth" source="AxisConfig" %}
 
 ### Title
 
-{% include table.html props="titleBaseline,titleColor,titleFont,titleLimit,titleFontWeight,titleFontSize,titlePadding,titleMaxLength" source="AxisConfig" %}
+{% include table.html props="maxExtent,minExtent,titleAlign,titleAngle,titleBaseline,titleColor,titleFont,titleLimit,titleFontWeight,titleFontSize,titlePadding,titleMaxLength" source="AxisConfig" %}
```
Comment (kanitw) [inline site/docs/axis.md]: maxExtent,minExtent,titleAlign,titleAngle are in VgAxisBase so they should be in Axis Properties too (in addition to AxisConfig)?

File: site/docs/legend.md [hunk 2]
```diff
@@ -65,19 +68,19 @@ Legend Config will apply to the all encoding channel has `legend`. Legend Config
 
 #### General
 
-{% include table.html props="cornerRadius,orient,offset,strokeColor,strokeDash,strokeWidth" source="LegendConfig" %}
+{% include table.html props="cornerRadius,fillColor,orient,offset,strokeColor,strokeDash,strokeWidth,padding" source="LegendConfig" %}
```
Comment (kanitw) [inline site/docs/legend.md]: padding should come before stroke (alphabetically)

File: site/docs/legend.md [hunk 3]
```diff
@@ -65,19 +68,19 @@ Legend Config will apply to the all encoding channel has `legend`. Legend Config
...
 
 #### Labels
 
 {% include table.html props="labelAlign,labelBaseline,labelColor,labelFont,labelFontSize,labelLimit,labelOffset,shortTimeLabels" source="LegendConfig" %}
 
 #### Symbols
 
-{% include table.html props="entryPadding,symbolColor,symbolSize,symbolStrokeWidth" source="LegendConfig" %}
+{% include table.html props="entryPadding,symbolColor,symbolType,symbolSize,symbolStrokeWidth" source="LegendConfig" %}
```
Comment (kanitw) [inline site/docs/legend.md]: sort alphabetically

File: site/docs/axis.md (hunk 1 above)
Comment (domoritz) [inline site/docs/axis.md]: They are.

File: site/docs/legend.md (hunk 3 above)
Comment (domoritz) [inline site/docs/legend.md]: done

File: site/docs/legend.md (hunk 2 above)
Comment (domoritz) [inline site/docs/legend.md]: done

**Pull request merged**